src/js/components/Calendar/Calendar.js

```diff
@@ -189,7 +189,34 @@ const Calendar = forwardRef(
     // clear the slide and targetDisplayBounds.
     useEffect(() => {
       const nextDisplayBounds = buildDisplayBounds(reference, firstDayOfWeek);
-      if (!animate) {
+
+      // Checks if the difference between the current and next DisplayBounds is
+      // greater than a year. If that's the case, calendar should update without
+      // animation.
```

Comment:
Thank you for adding this helpful comment and making the open-source code more readable.